crates/rspack_plugin_javascript/src/utils/eval/eval_lit_expr.rs

```diff
@@ -31,18 +48,20 @@ pub fn eval_lit_expr(expr: &Lit) -> Option<BasicEvaluatedExpression> {
       Some(res)
     }
     Lit::Num(num) => Some(eval_number(num)),
-    // TODO:
-    _ => None,
+    Lit::Bool(bool) => Some(eval_bool(bool)),
```

Comment:
Rspack previously leveraged `dead_branch_remover` and `optimizer` to evaluate `test` branch and remove dead branches. Without `run_before_pass`, we have to eval it in `ConstPlugin`.